README.md

```diff
@@ -1 +1,35 @@
-# edu-errors-ts-code
\ No newline at end of file
+# edu-errors-typescript-code
+
+This repository provides code used for exercises and demonstrations
+included in the TypeScript version of the
+[Crafting an Error Handling Strategy with TypeScript](https://learn.temporal.io/courses/tbd)
+training course.
+
+It's important to remember that the example code used in this course was designed to support learning a specific aspect of Temporal, not to serve as a ready-to-use template for implementing a production system.
+
+For the exercises, make sure to run `temporal server start-dev --ui-port 8080 --db-filename clusterdata.db` in one terminal to start the Temporal server. For more details on this command, please refer to the `Setting up a Local Development Environment` chapter in the course. Note: If you're using the Gitpod environment to run this exercise, you can skip this step.
```

Comment:
Consider breaking these down into bulleted prerequisites.

* Complete [Setting up a Local Development Environment](...) chapter
* Unless you're using Gitpod, ensure the Temporal CLI server is running on port 8080:

  ```
  temporal server start-dev \
    --ui-port 8080 \
    --db-filename temporal.db <-- notice how this does not use "cluster" or "data"
  ```

etc